src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -2,76 +2,43 @@
 
 
 import by.exadel.internship.dto.internshipDTO.GuestInternshipDTO;
-import by.exadel.internship.dto.LocationDTO;
-import by.exadel.internship.dto.enums.InternshipFormat;
-import by.exadel.internship.dto.enums.Technology;
+import by.exadel.internship.service.InternshipService;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
-import java.time.LocalDate;
-import java.util.ArrayList;
 import java.util.List;
+import java.util.UUID;
 
+@CrossOrigin
```

Comment:
To my mind, It's better to add global CORS configuration.
https://www.baeldung.com/spring-cors#global-cors-configuration

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/113557523-cf7d5600-9606-11eb-907e-32788556f0fc.png)